config/config.py

```diff
@@ -112,6 +118,10 @@ def build_config():
         ],
     }
 
+    config["producer_mods"] = [
```

Comment:
I would stick with writing out all names, so 

RemoveProducer instead of Prod_Remove
AppendProducer instead of Prod_Append

Reply:
also `producer_modifications` instead of `producer_mods`